src/pages/Account/Account.styles.tsx

```diff
@@ -0,0 +1,264 @@
+import { ExternalLink } from 'react-feather'
+import { Flex, Link, Text } from 'rebass'
+import styled from 'styled-components'
+
+import { ButtonPurpleDim } from '../../components/Button'
+import Logo from '../../components/Logo'
+import { ListLayout } from '../../ui/ListLayout'
+
+export const Status = styled(Flex)<{ status: string }>`
+  text-transform: uppercase;
+  padding: 4px;
+  font-size: 9px;
+  border-radius: 6px;
+  font-weight: 700;
+  border-width: 1.5px;
+  ${({ status }) => {
+    switch (status) {
+      case 'COMPLETED':
+      case 'CONFIRMED':
+      case 'CLAIMED':
+        return `
+        border-color: #0e9f6e;
+        color: #0e9f6e;`
+      case 'REDEEM':
+      case 'PENDING':
+        return `
+        border-color: #F2994A;
+        color: #F2994A;`
+      case 'CANCELLED':
+        return `
+        border-color: #F02E51;
+        color: #F02E51;`
+      default:
+        return `
+        border-color: #a7a0e4;
+        color: #a7a0e4;`
+    }
+  }};
```

Comment:
Should we declare this values inside a status dictionary and call it here?

Reply:
Done 👍🏼